<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebandDialog</class>
 <widget class="QDialog" name="DebandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deband effect settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="avgdiff">
     <property name="title">
      <string>Avgdiff</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QDoubleSpinBox" name="avgdiffValue">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>3.400000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="maxdiff">
     <property name="title">
      <string>Maxdiff</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QDoubleSpinBox" name="maxdiffValue">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>6.800000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="middiff">
     <property name="title">
      <string>Middiff</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QDoubleSpinBox" name="middiffValue">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>3.300000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="range">
     <property name="title">
      <string>Range</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QDoubleSpinBox" name="rangeValue">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>16.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="iterations">
     <property name="title">
      <string>Iterations</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QSpinBox" name="iterationsValue">
        <property name="minimum">
         <number>-10000</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DebandDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DebandDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
